blacktie/blacktie-admin-services/ear/pom.xml

```diff
@@ -76,6 +76,7 @@
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-ear-plugin</artifactId>
+        <version>2.5</version> <!-- this version places the dependency jars in the lib folder which is the right place -->
```

Comment:
This and the following changes are because the updated maven-ear-plugin does not generate the ear the same way (the jars are not placed in a lib/ folder but just in the root where they can't access each other

Reply:
I find [customizing-module-location](https://maven.apache.org/plugins/maven-ear-plugin/examples/customizing-module-location.html) in the maven-ear-plugin document. And it looks like we should use the **defaultLibBundleDir** to replace ```defaultJavaBundleDir```